integ/scripts/bash/run-e2e-automated.sh

```diff
@@ -30,19 +32,30 @@ refreshcreds() {
     unset CREDS
 }
 
+ASSUME_ROLE="false"
```

Comment:
A capitalized variable like this is used to denote an environment variable. If it isn't being exported, it should be lowercase.

Also, it you're using the string `"false"` instead of the boolean `false`, which I guess is fine, but not really what the intended purpose is.